src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/matrix_nms/matrix_nms_kernel_ref.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "kernel_base_opencl.h"
+#include "kernel_selector_params.h"
+
+namespace kernel_selector {
+
+enum decay_function { GAUSSIAN, LINEAR };
```

Comment:
Are you sure that these enums should be here not inside matrix_nms_params? Or maybe at least they should have "matrix_nms_" prefix?

Reply:
done